### ISSUE TITLE
multi scale training

### ISSUE THREAD

**Issue by kk1153** [Microsoft/singleshotpose#4]
currently the train code is only working with the size 416 X 416. 
what i see from the code is that it should change the scales based on the number of seen example in listDataset but that never happens. did i miss something ?

**Comment by btekin** [comment]
In [https://github.com/Microsoft/singleshotpose/blob/master/dataset.py#L41:L64](url), the resolution of the input image is changed randomly. In initial iterations, it starts with working on only 416x416, but as the iterations proceed, it starts to do multi-scale training. For how many epoch are you training the network? And, how do you initialize the network?

**Comment by kk1153** [comment]
I am training for default 700 epochs and it does not change the size at any stage. The problem i observed was it was never going into any elif condition because for each epoch, the value of seen reads as 0  from model.seen at start of each epoch. This model.seen was not getting updated after visiting the traindataloader. I added below in train_epoch function and then it does change the size based on number of example seen by model during training.

model.seen = model.seen + data.data.size(0)

I am initializing the network from darknetweights you have provided in as "darknet19_448.conv.23". I then do pretraining using yolo-pose-pre.cfg and then use the weights which give me lowest training loss to train the final network with yolo-pose.cfg

**Comment by btekin** [comment]
Thanks for pointing out this issue, now I fixed train.py by adding model.seen = model.seen + data.data.size(0) so that number of seen examples gets updated correctly.
